src/jdk.hotspot.agent/share/classes/sun/jvm/hotspot/SALauncher.java

```diff
@@ -100,6 +100,7 @@ private static boolean debugdHelp() {
         System.out.println("    --registryport <port>   Sets the RMI registry port." +
                 " This option overrides the system property 'sun.jvm.hotspot.rmi.port'. If not specified," +
                 " the system property is used. If the system property is not set, the default port 1099 is used.");
+        System.out.println("    --disableregistry       Do not start RMI registry (to use RMI registry that exists)");
```

Comment:
"to use RMI registry that exists" doesn't read well. Do you mean "use already started RMI registry"?